dask/array/core.py

```diff
@@ -4367,7 +4367,7 @@ def broadcast_shapes(*shapes):
     return tuple(reversed(out))
 
 
-def elemwise(op, *args, **kwargs):
+def elemwise(op, *args, out=None, where=True, dtype=None, name=None, **kwargs):
     """Apply elementwise function across arguments
 
     Respects broadcasting rules
```

Comment:
Can we update these docstrings and examples to show how `where` works?

Reply:
I've updated the docstring. `where` matches the numpy behavior and is rarely used, figure it's not worth adding a contrived example.

Reply:
I agree it's rarely used, but I do find it useful surprisingly often for avoiding divide-by-zero issues